### PR TITLE
Blog/building chrometober

### PR DESCRIPTION
**Pull request by jh3y**
Changes proposed in this pull request:

- "Building Chrometober" 🎃  
- Adds the hero for "Is it Modal"

### CODE REVIEW

File: src/site/content/en/blog/building-chrometober/index.md [hunk 1]
```diff
@@ -0,0 +1,1013 @@
...
+The goal of the project was to deliver a whimsical experience highlighting the scroll-linked animations API. But, whilst being whimsical, the experience needed to be responsive and accessible too. The project has also been a great way to test drive the API polyfill that is in active development; that, as well as trying different techniques and tools in combination. And all with a festive Halloween theme!
+
+Our team structure looked like this:
+
+- [Tyler Reed](https://twitter.com/intent/follow?screen_name=rudepetsclub) – Illustration & Design
+- [Jhey Tompkins](https://twitter.com/intent/follow?screen_name=jh3yy) – Architectural and Creative Lead
+- [Una Kravets](https://twitter.com/intent/follow?screen_name=una) – Project Lead
+- [Bramus Van Damme](https://twitter.com/intent/follow?screen_name=bramus) – Site Contributor
+- [Adam Argyle](https://twitter.com/intent/follow?screen_name=argyleink) – Accessibility Review
+- Aaron Forinton – Copywriting
```
Comment (rachelandrew) [inline src/site/content/en/blog/building-chrometober/index.md:39]:
```suggestion
- [Tyler Reed](https://twitter.com/intent/follow?screen_name=rudepetsclub): Illustration and design
- [Jhey Tompkins](https://twitter.com/intent/follow?screen_name=jh3yy): Architectural and creative lead
- [Una Kravets](https://twitter.com/intent/follow?screen_name=una): Project lead
- [Bramus Van Damme](https://twitter.com/intent/follow?screen_name=bramus): Site contributor
- [Adam Argyle](https://twitter.com/intent/follow?screen_name=argyleink): Accessibility review
- Aaron Forinton: Copywriting
```

File: src/site/content/en/blog/building-chrometober/index.md [hunk 2]
```diff
@@ -0,0 +1,1013 @@
...
+</li>
+```
+
+This code is mostly for setting up structure. Contributors can work on the book’s content for the most part without having to touch this code.
+
+### Backdrops
+
+The creative shift towards a book made splitting up the sections much easier, and each spread of the book is a scene taken from the original design.
+
+{% Img src="image/Dyx9FwYgMyNqy1kMGx8Orz6q0qC3/N3UtScgJr16t0Xp4vDN7.png", alt="Page spread illustration from the book which features an apple tree in a graveyard. The graveyard has multiple headstones and there is a bat in the sky in front of a large moon. ", width="800", height="560" %}
```
Comment (rachelandrew) [inline src/site/content/en/blog/building-chrometober/index.md:472]:
```suggestion
{% Img src="image/Dyx9FwYgMyNqy1kMGx8Orz6q0qC3/N3UtScgJr16t0Xp4vDN7.png", alt="Page spread illustration from the book which features an apple tree in a graveyard. The graveyard has multiple headstones and there is a bat in the sky in front of a large moon.", width="800", height="560" %}
```

**Review by rachelandrew — APPROVED** [review]
Just a couple of tiny things there and you are good to go.

**Comment by jh3y** [comment]
Thanks for catching them @rachelandrew 🙏

**Pull request merged**